src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -54,8 +54,8 @@ public function __construct(array $defaultClasses, array $options)
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $sonataAdmin = [
-            'name' => null,
-            'admin' => null,
+            'name' => false,
```

Comment:
You should revert this change @VincentLanglet 

This line is breaking with a `false` value:
https://github.com/sonata-project/SonataAdminBundle/blob/0639d128cb0516584ab435c0a726dd205ded8ef9/src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig#L100-L101

Reply:
Shouldn't we change the check to 
```
{% if sonata_admin.admin %}
```
instead ?

Either false or null shouldn't go in the if.
And we were mixing both null and false value before. I tried to always use `false` instead.
```
{% if sonata_admin.admin is not false %}
```
may work now but 
```
{% if sonata_admin.admin %}
```
might be safer and more logic

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/7609